src/gbd_mapping_generator/data.py

```diff
@@ -301,7 +301,8 @@ def get_risk_data():
             levels = sorted([(cat, name) for cat, name in risk['category_map'].items()],
                             key=lambda x: int(x[0][3:]))
             max_cat = int(levels[-1][0][3:]) + 1
-            levels.append((f'cat{max_cat}', 'unexposed'))
+            if rei_id not in contain_tmrel:
```

Comment:
ok this is a (super) nitpick-y thing but it's really fast to change and I like consistency so I'm going to say it: can we use 'Exposed' and 'Unexposed' since the category names we pull from GBD are capitalized? I think it's just a change on line 305 and lines 294-295. I double-checked in vph and it doesn't look like we ever use the category names themselves so it shouldn't break anything there

Reply:
haha I will change it. I like consistency also and VPH and anywhere else should follow it once we change it (and they only used cat, instead of name)

Reply:
so happy rn :100: